src/devices.rs

```diff
@@ -108,11 +108,39 @@ where
         }
     }
 
-    pub fn pixel_rgb(&mut self, x: u8, y: u8, r: u8, g: u8, b: u8) -> Result<(), Error<I2cError>> {
+    pub fn pixel_rgb_blocking(
+        &mut self,
+        x: u8,
+        y: u8,
+        r: u8,
+        g: u8,
+        b: u8,
+    ) -> Result<(), Error<I2cError>> {
         let x = (4 * (3 - x)) + y;
-        self.device.pixel(x, 0, r)?;
-        self.device.pixel(x, 1, g)?;
-        self.device.pixel(x, 2, b)?;
+        self.device.pixel_blocking(x, 0, r)?;
+        self.device.pixel_blocking(x, 1, g)?;
+        self.device.pixel_blocking(x, 2, b)?;
+        Ok(())
+    }
+}
+
+#[cfg(feature = "keybow_2040")]
```

Comment:
Since the `impl` block uses `embedded_hal_async`, this should also be gated by the `async` feature. Same with the other two affected devices.

```suggestion
#[cfg(all(feature = "keybow_2040", feature = "async"))]
```

Reply:
Done :)